chalk/shapes/path.py

```diff
@@ -1,122 +1,83 @@
 from __future__ import annotations
 
-import math
 from dataclasses import dataclass
-from typing import Any, List, Tuple, Union
+from typing import Any, Iterable, List, Tuple
 
 from chalk import transform as tx
 from chalk.envelope import Envelope
-from chalk.shapes.arc import ArcSegment
-from chalk.shapes.segment import Segment
 from chalk.shapes.shape import Shape
 from chalk.trace import Trace
+from chalk.trail import Located, Trail
 from chalk.transform import P2
-from chalk.types import Diagram
+from chalk.types import Diagram, Enveloped, Traceable
 from chalk.visitor import A, ShapeVisitor
 
 
 def make_path(segments: List[Tuple[float, float]]) -> Diagram:
     return Path.from_list_of_tuples(segments).stroke()
 
 
-SegmentLike = Union[Segment, ArcSegment]
-
-
 @dataclass
-class Path(Shape, tx.Transformable):
+class Path(Shape, Enveloped, Traceable):
```

Comment:
Shouldn't the `Path` be still `Transformable`?

Reply:
Good catch. Fixed.